api/base.go

```diff
@@ -28,7 +31,10 @@ func DefaultBaseConfig(
 		ClusterType:               clusterType,
 		CloudProviderType:         cloudProviderType,
 		EnvironmentName:           clusterName,
-		DNSPrefix:                 clusterName,
+		OIDCIssuerURL:             "https://dex." + clusterName + ".<domain name>",
```

Comment:
Should the default be this? Could it not make more sense to default to empty string? Or `change-me`

Reply:
Maybe you're right that this is a bit too specific. Somewhere we need to say that this is what you should set. But maybe that belongs in the ck8s repo with the other elastisys defaults.

Reply:
Yea, I think that would make more sense